sparse_strips/vello_hybrid/shaders/sparse_strip_renderer.wgsl

```diff
@@ -17,6 +17,9 @@ struct Config {
     height: u32,
     // Height of a strip in the rendering
     strip_height: u32,
+    // Number of trailing zeros in alphas_tex_width (log2 of width).
+    // Pre-calculated on CPU since WebGL2 doesn't support firstTrailingBit.
+    alphas_tex_width_bits: u32,
```

Comment:
Beyond the issue of WebGL2 not supporting `firstTrailingBit`. Uniforms in WebGL2 need to be 16 byte aligned.